### ISSUE TITLE
Labeling of preequilibration and simulation conditions

### ISSUE THREAD

**Issue by yannikschaelte** [Benchmarking-Initiative/Benchmark-Models-PEtab#52]
Can one use more informative names than `model1_data{1,2,3}`?

**Comment by LeonardSchmiester** [comment]
One can. But would require a lot of manual work as some models have hundreds of simulation conditions...

**Comment by yannikschaelte** [comment]
Agreed :/ If it is as simple as for the Zheng_PNAS2012 (done), Weber_BMC2015 models to distinguish preequilibration and simulation condition, it might be good to do so. But in general maybe not feasible.

**Comment by JanHasenauer** [comment]
I also think that more informative labels would be helpful.
For models which are in D2D, the respective dataset names could be used as a basis. This would also be a good basis for the datasetID.
